README.md

```diff
@@ -20,40 +20,40 @@ Make sure to check the Settings Descriptions below, especially to auto-start the
 
 ## Settings Descriptions
 
-Settings are found in the `settings.ini` file. Do not rename that file.
+Settings are found in the `settings.ini` file. Do not rename that file. It should be located with your executable file (e.g., `OpenVR-Dynamic-Resolution.exe`).
```

Comment:
```suggestion
Settings are found in the `settings.ini` file. Do not rename that file. It should be located in the same folder as your executable file (`OpenVR-Dynamic-Resolution.exe`).
```
User shouldn’t change the name of the executable, or else auto-start with SteamVR won’t work